docs/spec/api.md

```diff
@@ -4379,7 +4379,7 @@ The following parameters should be specified on the request:
 ###### On Success: Data Accepted
 
 ```
-204 No Content
+202 Accepted
```

Comment:
We need to fix all status codes in the descriptors and then refresh the API spec. For example, this one is located [here](https://github.com/distribution/distribution/blob/ad8f5caba068b2d4b83ba5a1c3e9d74e1da2c16e/registry/api/v2/descriptors.go#L1240).